### PR TITLE
Fix unique id usage

### PR DESCRIPTION
**Pull request by rainemak**
Do not generate unique ids rather use the uniqueID of the
webview.

### CODE REVIEW

**Comment by rainemak** [comment]
@chriadam review ?

File: import/WebView.qml [hunk 1]
```diff
@@ -156,7 +156,7 @@ RawWebView {
                         privData.contextMenu.linkTitle = linkTitle.trim()
                         privData.contextMenu.linkProtocol = data.linkProtocol || ""
                         privData.contextMenu.contentType = contentType
-                        privData.contextMenu.viewId = privData.uniqueId()
+                        privData.contextMenu.viewId = webView.uniqueID()
```
Comment (chriadam) [inline import/WebView.qml]: what defines this one?  QuickMozView?

**Comment by rainemak** [comment]
Correct, defined in QuickMozView.

**Comment by chriadam** [comment]
LGTM

**Pull request merged**